style.css

```diff
@@ -83,11 +85,42 @@ form {
 }
 
   #messages li { padding: 5px 10px; }
-    
+
     #messages li:nth-child(odd) { background: #eee; }
 
 
 /* == Individual messages == */
-.time { color: #2c3e50; }
+.time { color: #9c9c9c; }
```

Comment:
This color is good (_more subtle_): http://www.colorhexa.com/9c9c9c
![color-9c9c9c](https://cloud.githubusercontent.com/assets/194400/21419880/37c7a614-c822-11e6-9268-7762dcc4c68d.png)

the old one was a bit "_heavy_": http://www.colorhexa.com/2c3e50
![color-2c3e50](https://cloud.githubusercontent.com/assets/194400/21419874/3068f030-c822-11e6-9847-0fd930fef95d.png)